<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartRecordData</class>
 <widget class="QDialog" name="StartRecordData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 20px;</string>
  </property>
  <widget class="QGroupBox" name="fixedGpsDataGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>701</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(223, 223, 223);
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="gpsSaveAsPushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 239, 231);
font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(33, 32, 156);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gpsSaveAsLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="reordDataBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>620</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(245, 239, 231);
font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(33, 32, 156);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Record Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="setLatLongFPTextLabel_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>305</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(24, 90, 219);
color: rgb(245, 239, 231);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Axis-X and Axis-Y Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="XYAxisSettingGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>711</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(223, 223, 223);
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="manualPlotXYRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Manual X-Y Axis</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="automaticPlotXYRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Automatic X-Y Axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>98</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Corbel&quot;;</string>
    </property>
    <property name="text">
     <string>The software decide to choose X-Y Axis automatically!</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="manualXYAxisSettingGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>711</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Corbel&quot;;
border-radius: 20px;
background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="AxisXLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis-X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="axisXFromLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="axisXToLineEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="meterLabel1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="meterLabel2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="AxisYLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis-Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="axisYFromLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="axisYToLineEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="setRangePushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 239, 231);
font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(33, 32, 156);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Set Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="minXLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>148</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxXLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>148</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>148</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis-X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>148</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis-Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="minYLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>148</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxYLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>148</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel1_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>148</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel1_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>148</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>620</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(245, 239, 231);
font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(33, 32, 156);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="setLatLongFPTextLabel_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>511</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>305</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Corbel&quot;;
background-color: rgb(24, 90, 219);
color: rgb(245, 239, 231);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Choose direction in order to save data on .txt file:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>fixedGpsDataGroupBox</zorder>
  <zorder>reordDataBtn</zorder>
  <zorder>XYAxisSettingGroupBox</zorder>
  <zorder>manualXYAxisSettingGroupBox</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>setLatLongFPTextLabel_3</zorder>
  <zorder>setLatLongFPTextLabel_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
